### PR TITLE
feat: added home page banners to UI

### LINKED ISSUE #9: Create new Homepage Learn web3 banner
## Create the new "Learn Web3" banner for the home page

image

It will be placed under the Hero section of the home page.

Notes: 
- As for now it doesn't have a mobile version, so don't needed for now.
- Create it on the `packages/ui` internal package with the storybook story for it
- No need to create the homepage in the `apps/academy` app yet.

### LINKED ISSUE #10: Create new Homepage "partner with Academy" banner
## Create the new "partner with Academy" banner for the home page

image

It will be placed after the "Learn web3" banner, on the home page.

Notes: 
- As for now it doesn't have a mobile version, so don't needed.
- Create it on the `packages/ui` internal package with the storybook story for it.
- No need to create the homepage in the `apps/academy` app yet.

### PR DESCRIPTION
**Pull request by peteruche21**
## Changes

- Learn Web 3 homepage section
- DD partner homepage section

## API Updates
null

### screenshots

issue9
issue10

### New Features 

added banner components
added andale mono font
added banner component stories

Fixes #10 
Fixes #9

### CODE REVIEW

File: packages/ui/src/components/Banners/PartnerBanner.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        <CardContent className="description">
+          <p>
+            Developer DAO is a community of thousands of web3 builders creating a better internet.
+            Join us and help educate the next generation of developers.
+          </p>
+        </CardContent>
+        <CardContent className=" flex justify-center">
+          <Button asChild className="button-rounded bg-[#44AF96] text-black">
+            <NextLink href={href}>reach out</NextLink>
+          </Button>
```
Comment (Markkos89) [inline packages/ui/src/components/Banners/PartnerBanner.tsx:29]: Same thought about wrapping the button with the nextlink (viceversa)

**Review by Markkos89 — CHANGES_REQUESTED** [review]
Great work here @peteruche21 !! Amazing :rocket: 

Thanks a lot again, let me know if any question regarding the comments! Feel free!

File: packages/ui/src/components/Banners/LearnWeb3Banner.tsx [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+        <CardContent className="description">
+          <p>
+            DeveloperDAO Academy offers learning resources to help you learn how to build web3
+            technologies.
+          </p>
+        </CardContent>
+        <CardContent className=" flex justify-center">
+          <Button asChild className="button-rounded bg-[#44AF96] text-black">
+            <NextLink href={href}>get started</NextLink>
+          </Button>
```
[the remark this answers is not in the document]
Comment (peteruche21) [reply packages/ui/src/components/Banners/LearnWeb3Banner.tsx:27]: LOL, I'm not sure either. i basically used the example [here](https://ui.shadcn.com/docs/components/button#link) please check and let me know.

File: packages/ui/src/components/Banners/LearnWeb3Banner.stories.ts [hunk 3]
```diff
@@ -0,0 +1,27 @@
+/* eslint-disable storybook/no-title-property-in-meta */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
```
[the remark this answers is not in the document]
Comment (peteruche21) [reply packages/ui/src/components/Banners/LearnWeb3Banner.stories.ts:2]: I'm not sure where it came from. i am just seeing it now, it appears to be on other stories too. maybe auto generated or sort.

File: packages/ui/src/components/Banners/LearnWeb3Banner.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (peteruche21) [reply packages/ui/src/components/Banners/LearnWeb3Banner.tsx:27]: okay, I tested it, and it is correct the way it was implemented, doing it the other way around (i.e. wrapping the button component with the NextLink component) will cause it not to render.

File: packages/ui/src/components/Banners/LearnWeb3Banner.stories.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (peteruche21) [reply packages/ui/src/components/Banners/LearnWeb3Banner.stories.ts:2]: okay, found the source.

image

but from the type `Meta`:

```ts
type Meta<TCmpOrArgs = Args> = TCmpOrArgs extends ComponentType<any> ? ComponentAnnotations<ReactRenderer, ComponentProps<TCmpOrArgs>> : ComponentAnnotations<ReactRenderer, TCmpOrArgs>;
```

it seems `title` is indeed a member of `ComponentAnnotations`

maybe need to adjust the `eslint rules` globally or ignore on specific files.

**Review by Markkos89 — APPROVED** [review]

**Pull request merged**